comp/api/api/apiimpl/internal/agent/agent.go

```diff
@@ -575,14 +572,6 @@ func getDiagnose(w http.ResponseWriter, r *http.Request, diagnoseDeps diagnose.S
 	}
 }
 
-// max returns the maximum value between a and b.
-func max(a, b int) int {
```

Comment:
Hello, small question are we sure that every functions that used this one, use the one in go standard library ?

Reply:
The only place where this function was called is [here](https://github.com/DataDog/datadog-agent/blob/995481df7560af99947a27c4ba9c25397d6ed542/comp/api/api/apiimpl/internal/agent/agent.go#L424).

And this line was also removed because the cardinality is not useful here. 

So we don't need the max function at all in this context.

Reply:
perfect thx